local-remote-execution/README.md

```diff
@@ -1,45 +1,65 @@
 # Local Remote Execution
 
-Nativelink's Local Remote Execution is a framework to build and rapidly iterate
+NativeLink's Local Remote Execution is a framework to build and rapidly iterate
 custom toolchain setups that are transparent, fully hermetic and reproducible
-across machines of the same system architecture.
+across machines of the same system architecture. When used in conjunction with Nix,
+Local Remote Execution recreates the remote execution environment in your local
+development environment. Working with LRE lets you seamlessly switch between remote
+and local builds while reusing the same cache.
 
-When used in conjunction with Nix, Local Remote Execution recreates the remote
-execution environment in your local development environment. This lets you
-seamlessly switch between remote and local builds while reusing the same cache.
+You must clone the repo to use Local Remote Execution. However, the simplest way to use Local Remote Execution today is to [install and start NativeLink with Cargo](https://github.com/TraceMachina/nativelink?tab=readme-ov-file#-installing-with-cargo).
+## Pre-Requisites
 
-## Demo
+
+- Bazel 7.0.0 or later or another build system that implements [Bazel's RBE](https://bazel.build/remote/rbe)
+- Rust 1.73.0 or later
+- Nix 2.19.0 or later
+
+
+## Setting up Local Remote Execution
 
 > [!Note]
-> Local Remote Execution currently only works on `x86_64-linux`.
+> Local Remote Execution has only been tested on `x86_64-linux`.
 
-First, create an OCI image containing with the toolchains:
+1. Enable a [Flake-based](https://nixos.wiki/wiki/Development_environment_with_nix-shell) Nix Shell within the NativeLink root directory.
 
+```shell
+nix develop
 ```
+2. Create an [OCI image](https://opencontainers.org/) containing with the toolchains:
```

Comment:
"containing the toolchains," unless this is a syntax I don't know

Reply:
yikes thanks.